CardScan/Classes/ScanBaseViewController.swift

```diff
@@ -521,7 +496,6 @@ import Vision
         let rect = CGRect(x: cx - width / 2.0, y: cy - height / 2.0, width: width, height: height)
         
         self.currentImageRect = rect
-        
```

Comment:
Can you remove whitespace changes from here and the few other places that they show up?